src/test/java/org/apache/commons/collections4/FluentIterableTest.java

```diff
@@ -415,13 +415,17 @@ public void isEmpty() {
     }
 
     @Test
-    public void size() {
+    public void ofNull() {
         try {
-            FluentIterable.of((Iterable<?>) null).size();
```

Comment:
This test case maybe the same as factoryMethodOf() 
line 125.

Reply:
cf [my reply on the JIRA issue](https://issues.apache.org/jira/browse/COLLECTIONS-737?focusedCommentId=16983996&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-16983996); I have removed FluentIterableTest.ofNull.